misc/toml-parser/src/main/java/io/ballerina/toml/internal/parser/TomlParserErrorHandler.java

```diff
@@ -65,7 +65,8 @@
             { ParserRuleContext.ARRAY_VALUE_LIST_END, ParserRuleContext.ARRAY_VALUE_START };
 
     private static final ParserRuleContext[] NUMERICAL_LITERAL = {ParserRuleContext.DECIMAL_INTEGER_LITERAL,
-            ParserRuleContext.DECIMAL_FLOATING_POINT_LITERAL};
+            ParserRuleContext.DECIMAL_FLOATING_POINT_LITERAL, ParserRuleContext.HEX_INTEGER_LITERAL,
+            ParserRuleContext.OCTAL_INTEGER_LITERAL, ParserRuleContext.BINARY_INTEGER_LITERAL};
```

Comment:
correct formatting should be
```suggestion
            ParserRuleContext.OCTAL_INTEGER_LITERAL, ParserRuleContext.BINARY_INTEGER_LITERAL };
```
add a space after { here also.